# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/PBackground.ipdl
# Commit: 84162d09eef8
# Full Hash: 84162d09eef858c42992fa6cc623284bdab1fd05
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-12 21:30:33
# Regressor Bug: 1351231
# File Overlap Count: 2
# Description:
#   Bug 1351231 - PFetch protocol declaration and implementation. r=dom-worker-reviewers,jesup
#   
#   Depends on D138813
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142436
# ==============================================================================

diff -r 5c3b5a384f2f -r 84162d09eef8 ipc/glue/PBackground.ipdl
--- a/ipc/glue/PBackground.ipdl	Thu Jan 12 15:09:44 2023 +0000
+++ b/ipc/glue/PBackground.ipdl	Thu Jan 12 15:09:45 2023 +0000
@@ -53,6 +53,7 @@
 include protocol PRemoteDecoderManager;
 include protocol PWebSocketConnection;
 include protocol PWebTransport;
+include protocol PFetch;
 
 include DOMTypes;
 include IPCBlob;
@@ -131,6 +132,7 @@
   manages PUDPSocket;
   manages PVerifySSLServerCert;
   manages PVsync;
+  manages PFetch;
 
 parent:
   // Only called at startup during mochitests to check the basic infrastructure.
@@ -321,6 +323,8 @@
 
   async PIPCClientCerts();
 
+  async PFetch();
+
 child:
   async PCache();
   async PCacheStreamControl();
